packages/contracts/ethpm.js

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
this file should be named `ethpm.json`.  it wouldn't work for me named as it is right now.